src/mods/chat.js

```diff
@@ -849,7 +849,7 @@ idrinth.chat = {
             idrinth.chat.join ( data.data );
             return;
         }
-        idrinth.core.alert ( idrinth.text.get ( "chat.message.loginFailMoment" ) );
+        idrinth.core.alert ( idrinth.text.get ( "chat.error.loginMoment" ) );
```

Comment:
Replaced, thank you. Didn't even show up in my text collection, think it got under the wheels somewhere, going to check the difference